components/choose_download_action_dialog.py

```diff
@@ -93,8 +90,12 @@ def accept(self):
         )
         super().accept()
 
-    def create_button_box(self, buttons):
-        button_box = QDialogButtonBox(buttons)
+    def create_button_box(self, both):
```

Comment:
This refactoring on this dialog isn't *really* on-topic for the PR, but it makes debugging in a mixed Python environment easier. Apparently, in later versions of python, you can't do an `__or__` operation on the QDialogButtonBox.* roles unless those classes were refactored upstream by the PySide2 folks. This change avoids the "or" (`|`) symbol completely by adding additional buttons one at time.